docs/extensions/extension_manifest.md

```diff
@@ -1,37 +1,47 @@
 ---
-title: Manifest Format
+title: Extension Manifest Format
 category: 62c5cb137dbdad00536291a6
 hidden: true
 ---
 
-Extension metadata is defined in the `PhylumExt.toml` file, using the [TOML]
-format. Every manifest file consists of the following sections:
+## Overview
 
- - [`name`] — Extension name
- - [`entry_point`] — Execution entry point
- - [`permissions`] — Permissions required for execution
-     - [`[[read]]`] — Required read path permissions
-     - [`[[write]]`] — Required write path permissions
-     - [`[[env]]`] — Required environment variable permissions
-     - [`[[run]]`] — Required process execution permissions
-     - [`[[net]]`] — Required network domain permissions
+Extension metadata is defined in the `PhylumExt.toml` file, using the [TOML]
+format. Manifest files consist of the following sections:
+
+- [`name`] — Extension name
+- [`description`] - Description of the extension
+- [`entry_point`] — Execution entry point
+- [`[permissions]`][perms] — Permissions required for execution
+  - [`read`][read] — Required read path permissions
+  - [`write`][write] — Required write path permissions
+  - [`env`][env] — Required environment variable permissions
+  - [`run`][run] — Required process execution permissions
+  - [`net`][net] — Required network domain permissions
```

Comment:
These are arrays, so double brackets were correct.

Reply:
These are [**key-value pairs**](https://toml.io/en/v1.0.0#keyvalue-pair)...which have no extra notation (even if the value is an array).
Single brackets are used to represent [**tables**](https://toml.io/en/v1.0.0#table).
Double brackets are used to represent [**arrays of tables**](https://toml.io/en/v1.0.0#array-of-tables).

The [cargo manifest documentation](https://doc.rust-lang.org/cargo/reference/manifest.html), already pointed to for reference, makes use of this same format.